prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -2280,9 +2301,21 @@ ssize_t rxr_pkt_init_compare_rta(struct rxr_ep *ep, struct rxr_op_entry *tx_entr
 	 * the following append the data to be compared
 	 */
 	data = (char *)pkt_entry->pkt + pkt_entry->pkt_size;
-	data_size = ofi_copy_from_iov(data, ep->mtu_size - pkt_entry->pkt_size,
-				      tx_entry->atomic_ex.comp_iov,
-				      tx_entry->atomic_ex.comp_iov_count, 0);
+
+	for (i = 0; i < tx_entry->atomic_ex.comp_iov_count; i++) {
```

Comment:
wrap this up in a utility function `efa_copy_hmem_from_iov` should be better.

Reply:
done